plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -290,6 +290,22 @@
               version="v1-rev24-1.22.0" />
       </libraryFile>
     </library>
+    
+    <library
+          id="clouddatastore"
+          group="clientapis"
+          name="%clouddatastore.name"
+          tooltip="%clouddatastore.tooltip"
+          siteUri="https://cloud.google.com/appengine/docs/admin-api/" >
+      <libraryDependency id="googlecloudcore" />
+      <libraryFile
+            javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.6.0/apidocs/index.html?com/google/cloud/datastore/package-summary.html">    
```

Comment:
There exists 0.19.0, to which there is a link from the home of https://googlecloudplatform.github.io/google-cloud-java/.